getting-started/display.rst

```diff
@@ -0,0 +1,361 @@
+
+Improving the image display
+===========================
+
+The display framework gives you control over the image display to help bring out image details.
+
+To make masked regions semi-transparent, so that underlying image features are visible, try:
+
+.. code-block:: python
+
+   display.setMaskTransparency(60)
+
+The ``setMaskTransparency`` method's argument can range from ``0`` (fully opaque) to ``100`` (fully transparent).
+
+You can also control the colorbar scaling algorithm with the display's ``scale`` method.
+Try an ``asinh`` stretch with the ``zscale`` algorithm for automatically selecting the white and black points:
+
+.. code-block:: python
+
+   display.scale("asinh", "zscale")
+
+Instead of an automatic algorithm like zscale (or ``minmax``) you can explicitly provide both a minimum (black) and maximum (white) value:
+
+.. code-block:: python
+
+   display.scale("asinh", -1, 30)
+
+.. _getting-started-display-mask-colors:
+
+Interpreting displayed mask colors
+==================================
+
+The display framework renders each plane of the mask in a different color (*plane* being a different bit in the mask).
+To interpret these colors you can get a dictionary of mask planes from the ``calexp`` and query the display for the colors it rendered each mask plane with.
+Run:
+
+.. code-block:: python
+
+   for maskName, maskBit in mask.getMaskPlaneDict().items():
+       print('{}: {}'.format(maskName, display.getMaskPlaneColor(maskName)))
+
+As an example, this result is:
+
+.. code-block:: text
+
+   DETECTED_NEGATIVE: cyan
+   CROSSTALK: None
+   INTRP: green
+   DETECTED: blue
+   UNMASKEDNAN: None
+   NO_DATA: orange
+   BAD: red
+   EDGE: yellow
+   SUSPECT: yellow
+   NOT_DEBLENDED: None
+   CR: magenta
+   SAT: green
+
+Footprints of detected sources are rendered in blue and the saturated cores of bright stars are drawn in green.
+
+.. tip::
+
+   Try customizing the color of a mask plane with the ``Display.setMaskPlaneColor`` method.
+   You can choose any `X11 color`_.
+   For example:
+
+   .. code-block:: python
+
+      display.setMaskPlaneColor('DETECTED', 'dodgerblue')
+      display.mtv(calexp)
+
+Getting the source catalog generated by processCcd.py
+=====================================================
+
+Besides the calibrated exposure (``calexp``), :command:`processCcd.py` also creates a table of the sources it used for PSF estimation as well as astrometric and photometric calibration.
+The dataset type of this table is ``src``, which you can get from the Butler:
+
+.. code-block:: python
+
+   src = butler.get('src', dataId={'filter': 'HSC-R', 'visit': 903334, 'ccd': 23})
+
+This ``src`` dataset is a ``SourceTable``, which is a table object from the ``lsst.afw.table`` module.
+
+We'll explore ``SourceTable``\ s more in a later tutorial, but you can check its length with Python's `len` function:
+
+.. code-block:: python
+
+   print(len(src))
```

Comment:
Provide a ballpark number to expect (with the caveat that it will be stack version dependent)?